passive/s3.js

```diff
@@ -0,0 +1,40 @@
+// S3 bucket finder by alishasinghania09@gmail.com 
+
+function scan(ps, msg, src) {
+    // populate some parameters which will be needed if s3 bucket url is present
+    var alertRisk = 1
+    var alertReliability = 4
```

Comment:
- This should be 3 at most, 4 is for the user to set. The scan rule can't confirm that the URL is an actual leakage, e.g. docs, examples.
 - `Reliability` → `Confidence`